src/engine/video/video.cpp

```diff
@@ -389,8 +405,7 @@ const std::string VideoEngine::CreateGLErrorString()
 
     if(error_string == nullptr)
         return ("Unknown GL error code: " + NumberToString(_gl_error_code));
-    else
-        return (char *)error_string;
+    return static_cast<char*>(error_string);
```

Comment:
You asked me to do that. It failed to compile, I suppose that the previous behavior was not correct. Do you know a fix?